src/components/sbb-alert-group/sbb-alert-group.e2e.ts

```diff
@@ -1,82 +1,103 @@
 import { waitForCondition } from '../../global/testing';
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { EventSpy, waitForLitRender } from '../../global/testing';
+import { SbbAlertGroup } from './sbb-alert-group';
+import { SbbButton } from '../sbb-button';
+import '../sbb-alert-group';
+import '../sbb-alert';
 
 describe('sbb-alert-group', () => {
+  let element: SbbAlertGroup;
+
   it('should handle events ond states on interacting with alerts', async () => {
     const alertGroupId = 'alertgroup';
     const accessibilityTitle = 'Disruptions';
     const accessibilityTitleLevel = '3';
 
     // Given sbb-alert-group with two alerts
-    const page = await newE2EPage();
-    await page.setContent(`
-  <sbb-alert-group id='${alertGroupId}' accessibility-title='${accessibilityTitle}' accessibility-title-level='${accessibilityTitleLevel}'>
-    <sbb-alert title-content='Interruption' href='www.sbb.ch'>First</sbb-alert>
-    <sbb-alert title-content='Interruption' href='www.sbb.ch'>Second</sbb-alert>
-  </sbb-alert-group>
-`);
-    const didDismissAlertSpy = await page.spyOnEvent(events.didDismissAlert);
-    const emptySpy = await page.spyOnEvent(events.empty);
+    element = await fixture(html`
+      <sbb-alert-group
+        id="${alertGroupId}"
+        accessibility-title="${accessibilityTitle}"
+        accessibility-title-level="${accessibilityTitleLevel}"
+      >
+        <sbb-alert title-content="Interruption" href="www.sbb.ch">First</sbb-alert>
+        <sbb-alert title-content="Interruption" href="www.sbb.ch">Second</sbb-alert>
+      </sbb-alert-group>
+    `);
+    const didDismissAlertSpy = new EventSpy(events.didDismissAlert);
+    const emptySpy = new EventSpy(events.empty);
 
     // When rendering initially
-    await page.waitForChanges();
+    await waitForLitRender(element);
 
     // Then two alerts should be rendered and accessibility title should be displayed
-    expect((await page.findAll('sbb-alert')).length).toBe(2);
-    const alertGroupTitle = await page.find('sbb-alert-group >>> .sbb-alert-group__title');
-    expect(alertGroupTitle.textContent).toBe(accessibilityTitle);
-    expect(alertGroupTitle.tagName).toBe(`H${accessibilityTitleLevel}`);
+    expect(element.querySelectorAll('sbb-alert').length).to.be.equal(2);
+    const alertGroupTitle = element.shadowRoot.querySelector('.sbb-alert-group__title');
+    expect(alertGroupTitle.textContent.trim()).to.be.equal(accessibilityTitle);
+    expect(alertGroupTitle.tagName).to.be.equal(`H${accessibilityTitleLevel}`);
 
     // When clicking on close button of the first alert
-    await (await page.find('sbb-alert >>> .sbb-alert__close-button-wrapper sbb-button')).click();
-    await page.waitForChanges();
+    const closeButton = element
+      .querySelector('sbb-alert')
+      .shadowRoot.querySelector('.sbb-alert__close-button-wrapper sbb-button') as SbbButton;
+
+    closeButton.focus();
```

Comment:
Why is focus needed?

Reply:
It’s ok, was just interested as I expected to have the focus automatically on the button